gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagementTaskStreamImpl.java

```diff
@@ -93,22 +91,22 @@ public DagTask next() {
       throw new RuntimeException("DagManagement not initialized in multi-active execution mode when required.");
     }
     try {
-      MultiActiveLeaseArbiter.LeaseAttemptStatus leaseAttemptStatus = null;
+      LeaseAttemptStatus leaseAttemptStatus = null;
```

Comment:
since they're part and parcel of the MALA abstraction, I'd personally keep these as static inner classes.  did you find a particular motivation to move them?

either way is not a big deal actually...

Reply:
The only motivation was that there were too many (inner) classes in one file, total 4.
and to access LeaseObtainedStatus, one would have to write `MultiActiveLeaseArbiter.LeaseObtainedStatus` while imo `LeaseAttemptStatus.LeaseObtainedStatus` makes more sense, because it is closer to `LeaseObtainedStatus` than to `MultiActiveLeaseArbiter`